### PR TITLE
Fix/email notification

### PR DESCRIPTION
**Pull request by monik182**
This PR solves issue #308

### CODE REVIEW

File: templates/customers/profile-edit.liquid [hunk 1]
```diff
@@ -218,7 +218,26 @@ window.onload = function() {
         }
 
     }
-}
+
+
+    //Hide Email Input
+    var email = document.querySelector('.section-profile-communications').querySelector('[type=\'text\']'),
```
Comment (CristinaSolana) [inline templates/customers/profile-edit.liquid]: Let's give these inputs a class name that we can use to select instead.

File: templates/customers/profile-edit.liquid [hunk 2]
```diff
@@ -218,7 +218,26 @@ window.onload = function() {
...
+        checkBox = document.querySelector('.section-profile-communications').querySelector('[type=\'checkbox\']');
```
Comment (CristinaSolana) [inline templates/customers/profile-edit.liquid]: Same, class name for selector instead. (remember to prefix class with `js-`

File: assets/styles/layout/_page-profile-edit.scss [hunk 3]
```diff
@@ -19,7 +19,7 @@
     margin-top: 2rem;
 
     input[type="checkbox"] {
-        margin-bottom: 0;
+        margin-bottom: -2;
```
Comment (CristinaSolana) [inline assets/styles/layout/_page-profile-edit.scss]: -2 is not a valid value for margin. Must have a unit. Also, seems like a magic number. What is the reasoning for setting this to this value?

Comment (CristinaSolana) [reply assets/styles/layout/_page-profile-edit.scss]: Set this back to 0. On the input use the helper class: `margin-bottom-collapse`

Comment (cristianPerez) [reply assets/styles/layout/_page-profile-edit.scss]: Sorry was my idea !

**Comment by monik182** [comment]
The margin-bottom-collapse class didn't help to align the checkbox and text, so I had to wrap both elements into a flex container and used the align-center-middle class to align them.

**Pull request merged**